.github/workflows/main.yml

```diff
@@ -139,7 +129,17 @@ jobs:
       with:
         name: metagraph_${{ matrix.alphabet }}_linux_x86
         path: metagraph/build/metagraph_${{ matrix.alphabet }}${{ matrix.with_avx == 'OFF' && '_noAVX' || '' }}
+        
+    - name: run unit tests
+      if: matrix.build_static == 'OFF'
+      run: |
+        export LD_LIBRARY_PATH="/usr/local/lib/:$LD_LIBRARY_PATH"
+        cd metagraph/build && ./unit_tests
 
+    - name: run integration tests
+      run: |
+        export LD_LIBRARY_PATH="/usr/local/lib/:$LD_LIBRARY_PATH"
+        cd metagraph/build && make check
```

Comment:
why did you have to move these steps?

Reply:
It helps with debugging. If any of the test segfault and I can't reproduce the issue locally, this allows me to download the binary and run that instead.